js/index.js

```diff
@@ -40,3 +40,81 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let ctaImg = document.querySelector('#cta-img')
+ctaImg.setAttribute('src', siteContent["cta"]["img-src"])
+
+let middleImg = document.querySelector('#middle-img')
+middleImg.setAttribute('src', siteContent["main-content"]["middle-img-src"])
+
+// Updating the NavMenu
+let navMenu = document.querySelectorAll('a');
+
+navMenu.forEach((value, index) => value.textContent = siteContent["nav"][`nav-item-${index+1}`])
+
+// Updating CTA
+
+let ctaHeader = document.querySelector('.cta-text h1')
+ctaHeader.textContent = siteContent['cta']['h1']
+
+let buttonText = document.querySelector('.cta-text button')
+buttonText.textContent = siteContent['cta']['button']
+
+// Updating top Content Headers
+
+let topTextHeaders = document.querySelectorAll('.top-content .text-content h4')
+topTextHeaders[0].textContent = siteContent['main-content']['features-h4']
+topTextHeaders[1].textContent = siteContent['main-content']['about-h4']
+
+// Updating top Content  Text
+let topText = document.querySelectorAll('.top-content .text-content p')
+topText[0].textContent = siteContent['main-content']['features-content']
+topText[1].textContent = siteContent['main-content']['about-content']
+
+// Updating Boittom Content Headers
+let bottomTextHeaders = document.querySelectorAll('.bottom-content .text-content h4')
+bottomTextHeaders[0].textContent = siteContent['main-content']['services-h4']
+bottomTextHeaders[1].textContent = siteContent['main-content']['product-h4']
+bottomTextHeaders[2].textContent = siteContent['main-content']['vision-h4']
+
+// Updating Bottom Content  Text
+let bottomText = document.querySelectorAll('.bottom-content .text-content p')
+bottomText[0].textContent = siteContent['main-content']['services-content']
+bottomText[1].textContent = siteContent['main-content']['product-content']
+bottomText[2].textContent = siteContent['main-content']['vision-content']
+
```

Comment:
good work with this code, you've written out a very direct way to get the job done, but consider how you could use functions to automate the same process. for example >> 
```
// make an array of the needed content
  function arrayzer(content, tag) {
    return Object
    .entries(siteContent[content])
    .filter(item => item[0].includes(tag))
    .map(item => item[1]);
  }

//populate the content areas >> main content
function fillContentCards() {
  const headerText = arrayzer('main-content', 'h4');
  let headers = document.querySelectorAll('.text-content h4');
  headers.forEach((e,i) => {
    e.textContent = headerText[i];
  });
  
  const contentText = arrayzer('main-content', 'content');
  let contentP = document.querySelectorAll('.text-content p');
  contentP.forEach((e,i) => {
    e.textContent = contentText[i];
  });
}
fillContentCards(); 
```